Client/Frontend/AuthenticationManager/AuthenticationKeychainInfo.swift

```diff
@@ -100,17 +100,30 @@ extension AuthenticationKeychainInfo {
     }
 
     func isLocked() -> Bool {
-        if NSProcessInfo().systemUptimeTimestamp() < self.lockOutTimestamp {
+        if SystemUtils.systemUptime() < self.lockOutInterval {
             // Unlock and require passcode input
             resetLockoutState()
```

Comment:
You're right - someone could just reset the phone and avoid the lock out timer. This doesn't seem that big of a problem becuase:

1. We actually don't tell the user how long they are locked out for - it just says try later. 
2. Restarting the phone still takes a bit of time. I think it takes long enough to prevent any kind of brute force attacks.

Ideally I'd like to find a better way of handling this but I can't think of any other notion of time we can use that the user can't reset or modify locally. It would be annoying to have to resort to using a server provided timestamp :/